guides/common/modules/proc_creating_an_application_definition.adoc

```diff
@@ -0,0 +1,30 @@
+[id="{context}_creating_an_application_definition"]
+= Creating an Application Definition
+
+Use this procedure to create xref:{context}_application_definitions[application definitions] to use as part of your application centric deployment.
+
+.Prerequisites
+You need existing host groups in order to use application centric deployment.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Applications > App Definitions*.
+. Click *New Application Definition*.
+. In the *Name* field, enter the name of the new application definition.
+. In the *Description* field, enter an arbitrary description.
+. From the *Ansible Playbook* list, select the name of an Ansible playbook.
+. Click the plus icon to add new sets of hosts.
+.. In the *Name* field, enter the name of the new set of hosts.
```

Comment:
@maximiliankolb here you have substeps rather than bullets and I think it looks good.